kieserver/image.yaml

```
@@ -130,6 +130,9 @@ envs:
     - name: "OPTAPLANNER_SERVER_EXT_THREAD_POOL_QUEUE_SIZE"
       example: "4"
       description: "The default queue size is equal to the number of active CPU cores. (Sets the org.optaplanner.server.ext.thread.pool.queue.size system property)"
+    - name: "KIE_SERVER_MAX_METASPACE_SIZE"
+      example: "256"
+      description: "The maximum metaspace for the KIE Server, it will set the GC_MAX_METASPACE_SIZE, its default value is 512mb."
```

Same suggestion as for PAM KIE Server image
```suggestion
      description: "The maximum metaspace size, set as integer in megabytes, for the KIE Server. This will overwrite value set by GC_MAX_METASPACE_SIZE. Default max metaspace size for KIE Server is 512mb."
```